at_secondary/at_persistence_secondary_server/lib/src/keystore/hive_keystore.dart

```diff
@@ -159,6 +164,12 @@ class HiveKeystore implements SecondaryKeyStore<String, AtData?, AtMetaData?> {
       String? sharedKeyEncrypted,
       String? publicKeyChecksum,
       String? encoding}) async {
+    final atKey = AtKey.getKeyType(key);
```

Comment:
Please explicitly pass enforceNamespace parameter, for code readability / maintainability and to prevent accidents in future if the default value changes to 'true;'

Reply:
done